cmd/minishift/cmd/start.go

```diff
@@ -274,11 +291,24 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Instance will be configured with ...")
+		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   vCPUs :   ", machineConfig.CPUs)
+		fmt.Println("   Disksize: ", units.HumanSize(float64(machineConfig.DiskSize*units.MB)))
+		fmt.Println("   Boot ISO: ", machineConfig.MinikubeISO)
```

Comment:
Agree with @hferentschik . It should show the local cached version.  However the other displayed information are nice.